draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -41,24 +41,28 @@ informative:
 
 --- abstract
 
-QUIC defines a RESET_STREAM frame to abort sending on a stream. When a sender
-resets a stream, it also stops retransmitting STREAM frames for this stream in
-the of packet loss. On the receiving side, there is no guarantee that any data
-sent on that stream is delivered.
+The QUIC transport protocol provides byte streams that are ordered and reliable.
+Once a sender has started to emit stream data, it can abort the operation using
+a RESET_STREAM frame. When a sender resets a stream, it also stops
+retransmitting stream data in the case of packet loss. At the receiver, there is
+no guarantee that any data sent on a reset stream is delivered.
 
-This document defines a new QUIC frame, the RESET_STREAM_AT frame, that allows
-resetting a stream, while guaranteeing delivery of stream data up to a certain
-byte offset.
+This document defines an extension that allows a sender to reset a stream with
+guaranteed delivery of stream data from the beginning of the stream up to an
+explicitly declared byte offset.
 
 --- middle
 
 # Introduction
 
-QUIC version 1 ({{!RFC9000}}) allows streams to be reset.  When a stream is
-reset, the sender doesn't retransmit stream data for the respective stream. On
-the receiving side, the QUIC stack is free to surface the stream reset to the
-application immediately, without providing any stream data it has received for
-that stream.
+Version 1 of the QUIC transport protocol ({{!QUIC=RFC9000}}) provides
+streams, an ordered and reliable byte stream. Once a sender has started to emit
```

Comment:
@marten-seemann if you are concerned about that, maybe something like:
```suggestion
byte-oriented streams that are ordered and reliable. Once a sender has started to emit
```

It's almost a duplicate of the first sentence of the abstract, but hopefully nobody cares.

Reply:
I merged too quickly. Byte-oriented? I've never heard this term before.
Maybe better: reliable byte-ordered streams.